docs/configuration.adoc

```diff
@@ -195,183 +195,219 @@ The following script levels are reserved by pre-defined scripts:
 This convention allows external configurations to customize the configuration during various phases of the initialization process. For example, a script named `01-remove-extras.sh` could delete unwanted drivers or plugins from the staging directory. A script named `76-set-passwords.groovy` could change the username and password of a data source.
 
 [[oak]]
-=== Oak Configuration
+=== inetsoft.yaml Configuration
 
-By default, Oak is configured using the internal database for the document node store and a file blob store with sensible file paths. If you want to use MongoDB for the document node store or a different blob store, you'll need to include a custom `config/oak-config.yaml` file in your external configuration.
+The configuration of the storage back end is done in inetsoft.yaml. This file also contains the audit database configuration, the Hazelcast cluster configuration, as well a couple of other top-level initialization settings.
```

Comment:
The key-value and blob store are probably the most important things configured in this file, they should be mentioned.